server/app/services/question/types/QuestionDefinition.java

```diff
@@ -24,6 +24,10 @@ public abstract class QuestionDefinition {
   private final QuestionDefinitionConfig config;
 
   protected QuestionDefinition(QuestionDefinitionConfig config) {
+    if (config.validationPredicates().isEmpty()) {
+      config = config.toBuilder().setValidationPredicates(getDefaultValidationPredicates()).build();
+    }
+
```

Comment:
Technically, we could remove the above and leave `validationPredicates()` as empty in `QuestionDefinitionConfig`, since `getValidationPredicates()` on line 176 is handling that case anyways. Any thoughts welcome.

Reply:
I think it's safe to remove.

Reply:
Removed.